src/CloudKit/CKSyncEngineSendChangesScope.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using ObjCRuntime;
+using Foundation;
+
+#if !NET
+using NativeHandle = System.IntPtr;
+#endif
+
+#nullable enable
+
+namespace CloudKit {
+	public partial class CKSyncEngineSendChangesScope {
+		public CKSyncEngineSendChangesScope (NSSet<CKRecordZoneID>? zoneIds, bool excluded)
```

Comment:
Question - the zoneIds set here is nullable, but I can't find any reference to null being a valid value in Apple's documentation. Since you never test it for null, why not just make it non-nullable?
If that's not OK, why not set it to an empty set if it's null?